src/lando/api/legacy/workers/landing_worker.py

```diff
@@ -47,14 +44,13 @@
 
 
 @contextmanager
-def job_processing(worker: LandingWorker, job: LandingJob):
```

Comment:
The signature is updated here but there are no changes to callers, guessing this belongs in a different PR.

Reply:
I'll add this in the commit message. It's an unused variable that I just removed while I was here.

Reply:
Also forgot to update the caller, which is now done. This was pulled from a larger PR earlier, but I think it makes more sense to include it in this one. In any case, the latest 3 PRs are closely related, the last in the chain being #108